### DISCUSSION TITLE
folder module

### DISCUSSION THREAD

**Discussion by btsimonh** [tasmota/Berry_playground#22] [Modules]
with path.rmdir and path.mkdir now available in the development branch of TAS, I've got a PR in for a simple example, and a zap(folder) function written in berry which deleted a folder and contents.

PR https://github.com/tasmota/Berry_playground/pull/21 Up for discussion here...

**Comment by dkebler** [comment]
is this a full on module or just an example/tutorial?  In the later case would go in tutorial/code blocks folder

**Comment by btsimonh** [comment]
good question.  the zap function is a general FS function, which could be used generally, and so could be considered a module.
I guess I'm unsure what we would describe as a module... :)

**Comment by btsimonh** [comment]
Note that th ZAP funtion could be very dangerous as is with proposed changes to Berry path.listdir...  so let's hold until that's sorted anyway :)
